### PR TITLE
Add GMT space

### PR DESCRIPTION
**Pull request by aurumminer**
**Add space for GMT**  
  
Etherscan: https://etherscan.io/address/0xF5f8472045c9177B545Ea05fdA73848291a3f8B8
Website: https://goldmining.farm 
https://twitter.com/goldminingfarm

### CODE REVIEW

**Review by aurumminer — COMMENTED** [review]
all ok.

File: spaces/homepage.json [hunk 1]
```diff
@@ -8,5 +8,6 @@
   "rarible",
   "pickle",
   "aavegotchi",
-  "uniswap"
+  "uniswap",
+  "gmt"
```
Comment (e-asia) [inline spaces/homepage.json:12]: The homepage will be edited by Snapshot team only, please remove `homepage.json` from your PR. Thank you.

**Review by e-asia — CHANGES_REQUESTED** [review]

File: spaces/gmt/skin.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.egg {
```
Comment (e-asia) [inline spaces/gmt/skin.scss:1]: The css selector should similar to the space key.
```suggestion
.gmt {
```

**Review by e-asia — CHANGES_REQUESTED** [review]

**Review by aurumminer — COMMENTED** [review]
updated.

**Review by e-asia — APPROVED** [review]

**Review by aurumminer — COMMENTED** [review]
seems ok

**Comment by aurumminer** [comment]
Thank you. Its integrated.

**Pull request merged**